final_prep/README.md

```diff
@@ -23,52 +23,52 @@ Some things to consider as you refactor include...
+- [X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+-  <img width="1333" alt="Screen Shot 2022-02-13 at 12 45 56 AM" src="https://user-images.githubusercontent.com/98673086/153744214-923961b7-8cac-4206-a3e5-0b1d76d96d0d.png">
+- <img width="1333" alt="Screen Shot 2022-02-13 at 12 46 08 AM" src="https://user-images.githubusercontent.com/98673086/153744218-ebf322a0-a074-468b-aa58-1f359a4c26e8.png">
+- <img width="1333" alt="Screen Shot 2022-02-13 at 12 46 22 AM" src="https://user-images.githubusercontent.com/98673086/153744227-267473c6-5122-48ec-98e8-8f88bd9f0f3d.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [X] [link](https://gist.github.com/lcole37/a942e8c1e83f105e0fa59748fab48824)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+- [X] [gist here](https://gist.github.com/lcole37/6992bae17e31765f35517e9316ce39d5)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [X] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+- [X] [link](https://gist.github.com/lcole37/69c586cb602d5a3dbe6925699769737b)
```

Comment:
Decent start here. Keep this gist handy going forward, and make updates each time you go through a new group DTR. You'll benefit by saving time of having to fill it out every time, and be able to make updates to understand what's working and what you might want to clarify with future teammates.